static/js/components/ClassificationList.jsx

```diff
@@ -116,6 +117,10 @@ const ClassificationList = () => {
       } else {
         taxname = "Unknown taxonomy";
       }
+      if (usedClassifications.includes(classification)) {
```

Comment:
It's ok to have the same classification if it's in a different (latest) taxonomy.

Reply:
Yeah but we wouldn't want to redisplay... right?